widgets/serviceButton/src/ServiceButton.tsx

```diff
@@ -1,36 +1,43 @@
 import type { SemanticCOLORS, SemanticICONS } from 'semantic-ui-react';
 import type { FunctionComponent } from 'react';
-import type { MarketplaceTab } from '../../common/src/blueprintMarketplace/types';
+
+const { Button } = Stage.Basic;
+const { drilldownPage } = Stage.Common.Consts;
 
 export interface ServiceButtonProps {
     basic: boolean;
     color: SemanticCOLORS;
     icon: SemanticICONS;
     label: string;
-    marketplaceTabs: MarketplaceTab[];
+    defaultMarketplaceTab?: string;
+    toolbox: Stage.Types.Toolbox;
 }
 
-const ServiceButton: FunctionComponent<ServiceButtonProps> = ({ basic, color, icon, label, marketplaceTabs = [] }) => {
-    const { useBoolean } = Stage.Hooks;
-    const [open, setOpen, unsetOpen] = useBoolean();
-
-    const { Button } = Stage.Basic;
-    const { Modal } = Stage.Common.BlueprintMarketplace;
+const ServiceButton: FunctionComponent<ServiceButtonProps> = ({
+    basic,
+    color,
+    icon,
+    label,
+    defaultMarketplaceTab,
+    toolbox
+}) => {
```

Comment:
maybe we could use drop FunctionComponent here and simple use destructured props like this?:
```suggestion
const ServiceButton = ({ basic, color, icon, label, defaultMarketplaceTab, toolbox }: ServiceButtonProps) => {
```
I think React.FC is discourged ref: https://github.com/facebook/create-react-app/pull/8177 & https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/function_components/

WDYT?

Reply:
Looking at our [UI Code Style Guideliness](https://cloudifysource.atlassian.net/wiki/spaces/RD/pages/435388492/UI+Code+Style#MUST%3A-Use-the-correct-type-for-component-props) for components declared as `const MyComponent = (props) => (...)` we should use `FunctionComponent` or `VoidFunctionComponent` when it comes to the component typing

But as `VoidFunctionComponent` seems to not be supported anymore (I even can't import it from the `react` package anymore 😅), maybe it would be a good point to have a discussion on, during one of our weeklies? 🤔 
It surely could lead to making our codebase more consistent 🎉

Reply:
sure, we can add this topic to discuss on our weeklies :) thanks for reply